dev-util/premake/premake-5.0.0_alpha16.ebuild

```diff
@@ -20,6 +21,15 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-5.0.0_alpha16-respect-MAKEOPTS-jobs.patch
 )
 
+src_prepare() {
+	default
+	if use prefix; then
+		grep -rl '\-L/usr' ${S} | while IFS= read srcfile; do
```

Comment:
How many affected files are there?  If it is less than 5.  Writing them out explicitly is more robust.

Reply:
Also, really, I'd prefer that we just patch this out, as it's wrong in other contexts anyway. It's usually not needed to specify it at all if the compiler knows the search paths.

Please only use seds like this as a last resort, as they're hard to test / validate.